### DISCUSSION TITLE
Sir, May i join team?

### DISCUSSION THREAD

**Discussion by Legendleo90** [ROM-builders/temporary#20061] [General]
GitHub : Legendleo90
Telegram : @leopubglover

Device tree : https://github.com/Legendleo90/device_xiaomi_beryllium
Common tree:
https://github.com/Legendleo90/device_xiaomi_sdm845-common

Successful build : https://cirrus-ci.com/build/5724951970316288
https://cirrus-ci.com/build/6561000908914688

**Comment by Apon77** [comment]
Please follow #learn instructions from telegram group
